chat_api_sample/build.gradle

```diff
@@ -6,30 +6,25 @@ android {
 
     defaultConfig {
         applicationId "com.zopim.sample.chatapi"
-        minSdkVersion 17
+        minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
         versionName "1.0"
-
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-
     }
     buildTypes {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
-
     android {
         dataBinding {
             enabled true
         }
     }
-}
-
-repositories {
-    maven { url 'https://zendesk.artifactoryonline.com/zendesk/repo' }
+    lintOptions {
+        abortOnError false
```

Comment:
`abortOnError` should always be `true`, can you please add a lint config if there are any errors